### PR TITLE
feat: 💡 mobile responsive nav bar

### PR DESCRIPTION
**Pull request by OyinloluB**
## Why?

Setting up mobile screen styles for nav bar

## How?

- Imported new hamburger icon from the `react-icons` library
- Conditionally displayed elements depending on screen size
- Modified styles for mobile screen sizes
- Hid search input for mobile screens

## Tickets?

- [Notion](https://www.notion.so/Nice-to-have-Mobile-responsiveness-7e7a516084ec41d9989b596d8111e83d)

## Contribution checklist?

- [x] The commit messages are detailed
- [x] It does not break existing features (unless required)
- [x] I have performed a self-review of my own code
- [ ] Documentation has been updated to reflect the changes
- [ ] Tests have been added or updated to reflect the changes
- [x] All code formatting pass
- [x] All lints pass

## Demo?

Screenshot 2022-07-06 at 01 58 50

https://user-images.githubusercontent.com/51888121/178258460-17dd2ef8-d28e-4a07-80d6-7253de96e77f.mov

https://user-images.githubusercontent.com/51888121/178258598-7d1ce209-7bac-44cb-b032-7e5525ffde0a.mov

### CODE REVIEW

File: src/components/plug/plug.tsx [hunk 1]
```diff
@@ -117,66 +114,6 @@ export const Plug = () => {
     }
   }, [isConnected, dispatch]);
 
-  const onConnectionUpdate = () => {
```
Comment (prasanthLalapeta) [inline src/components/plug/plug.tsx:120]: I'm not sure why we removed this code. If something not related to mobile responsiveness navbar, can we move to seperate PR

Comment (OyinloluB) [reply src/components/plug/plug.tsx:120]: It was moved to a seperate file so it can be reused by the mobile navbar component @prasanthLalapeta

Comment (prasanthLalapeta) [reply src/components/plug/plug.tsx:120]: Ok

File: src/components/plug/plug-balance.tsx [hunk 2]
```diff
@@ -59,8 +58,8 @@ const PlugBalance = () => {
         src={wicpImage}
         alt={t('translation:logoAlts.wicp')}
       />
-      {plugWicpBalance !== '' && !plugLoadingWicpBalance ? (
-        `${roundOffDecimalValue(Number(plugWicpBalance), 2)}`
+      {wicpBalance !== '' && !loadingWicpBalance ? (
```
[the remark this answers is not in the document]
Comment (OyinloluB) [reply src/components/plug/plug-balance.tsx:61]: Yes from what we have in the develop branch, this is different. I moved the `wicpBalance` into redux to be used across components.

On the develop, we only have the state in this component so I moved it into redux.

**Review by prasanthLalapeta — APPROVED** [review]

**Pull request merged**